src/API.h

```diff
@@ -0,0 +1,101 @@
+//
+//  API.h
```

Comment:
Agree it is in PR comment

On Thu, Jun 16, 2016, 21:08 Kyle Fuller <notifications@github.com> wrote:

> In src/API.h
> <https://github.com/apiaryio/drafter/pull/341#discussion_r67405921>:
>
> > @@ -0,0 +1,101 @@
> > +//
> > +//  API.h
>
> I think we should call this header drafter.h so people will include
> drafter.h and not API.h when using this library. This will be clearer and
> consistent with other C/C++ libraries.
>
> Examples:
>
>    - Curl (#include <curl/curl.h>)
>    - gnutls (#include <gnutls/gnutls.h>)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apiaryio/drafter/pull/341/files/5aa7fe394ab8f51fb8fd2b1b778459c911f1f6c0#r67405921>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAd59fHKmR33UBlqhDoHLZ6CJcc8mIUWks5qMZ8rgaJpZM4I3DDc>
> .
>